docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -895,6 +895,537 @@ Total number of buffer pool classes loaded since the JVM started.
 (integer)
 Total number of buffer pool classes unloaded since the JVM started.
 
+[[cluster-nodes-stats-api-response-body-threadpool]]
+===== `threadpool` section
```

Comment:
Should this be `thread_pool` to match the values? I only bring this up because the section title is in backticks.

Reply:
Good catch. Fixed with e079d8f.